src/react/components/pages/editorPage/canvas.test.tsx

```diff
@@ -12,10 +12,19 @@ import Canvas, { ICanvasProps, ICanvasState } from "./canvas";
 import CanvasHelpers from "./canvasHelpers";
 
 jest.mock("vott-ct/lib/js/CanvasTools/CanvasTools.Editor");
+import { Editor } from "vott-ct/lib/js/CanvasTools/CanvasTools.Editor";
 
 jest.mock("vott-ct/lib/js/CanvasTools/Region/RegionsManager");
+import { RegionsManager } from "vott-ct/lib/js/CanvasTools/Region/RegionsManager";
 
 describe("Editor Canvas", () => {
+
+    function createTestRegionData() {
```

Comment:
So it looks like you aren't using this method, but are using the one defined in mock factory, so you can probably get rid of this